package.json

```diff
@@ -1,44 +1,22 @@
 {
-  "name": "@madebywild/eslint-config",
-  "version": "0.2.2",
-  "description": "wild's ESLint config",
-  "homepage": "https://github.com/madebywild/eslint-config",
-  "author": "WILD <weare@wild.as> (https://wild.as)",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/madebywild/eslint-config/issues"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/madebywild/eslint-config.git"
-  },
+  "name": "@madebywild/eslint-configs",
```

Comment:
I guess the plural is needed because we now have a react-config that is named `eslint-config`? Maybe we should consider renaming that one to `eslint-config-react`? cc @lorenzomigliorero

Reply:
oh well, the main config isn’t used, it is just a monorepo requirement. The name can be simply changed as `eslint-config`, no matter if it's equal to others packages.
I would prefer to use just `@madebywild/eslint-config` for the main config, as other popular libs.
Otherwise, we’ll have another `@madebywild/eslint-*` distributed on NPM.